companies/american-alpine-club.md

```diff
@@ -0,0 +1,18 @@
+---
+name: American Alpine Club
+image: american-alpine-club.png
+blm_statements:
+  - url: https://twitter.com/americanalpine/status/1267946389482266625
+    date_posted: 2020-06-02
+    type: Tweet
+    raw_text: >
+    Black lives matter—throughout the country and at the crag. The AAC stands in solidarity with those protesting systemic racism on the frontlines and from home.
```

Comment:
```suggestion
      Black lives matter—throughout the country and at the crag. The AAC stands in solidarity with those protesting systemic racism on the frontlines and from home.
```

Reply:
I was able to fix the errors, acceptable/safe to merge or no?